addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3599,18 +3614,6 @@ const SnippetOptionWidget = Widget.extend({
                 const _restoreTransitions = () => this.$target[0].classList.remove('o_we_force_no_transition');
 
                 const styles = window.getComputedStyle(this.$target[0]);
-
-                if (params.withGradients && params.cssProperty === 'background-color') {
-                    // Check if there is a gradient, in that case this is the
-                    // value to be returned, we normally not allow color and
-                    // gradient at the same time (the option would remove one
-                    // if editing the other).
-                    const parts = backgroundImageCssToParts(styles['background-image']);
-                    if (parts.gradient) {
-                        return parts.gradient;
-                    }
-                }
-
```

Comment:
It was not moved in order to happen after `_restoreTransitions`, but indeed that move should be a fix, otherwise we returned before calling it, and the `o_we_force_no_transition` class is not removed. (reproduced on master - will create a fix after hunting version down)
The move was done because instead of returning the gradient like before, we now replace value - so it has to happen after value is defined (to avoid duplicating the next part of the logic), so that in case the gradient is coming from the "cc", it gets turned into an empty string.